common/utils.js

```diff
@@ -202,13 +202,14 @@ export function getMedianValue(array) {
 // Set tf.js backend based WebNN's 'MLDeviceType' option
 export async function setPolyfillBackend(device) {
   // Simulate WebNN's device selection using various tf.js backends.
-  // MLDeviceType: ['default', 'gpu', 'cpu']
-  // 'default' or 'gpu': tfjs-backend-webgl, 'cpu': tfjs-backend-wasm
-  if (!device) device = 'gpu';
+  // MLDeviceType: ['default', 'webgl', 'webgpu', 'cpu']
+  // 'default' or 'webgl': tfjs-backend-webgl, 'webgpu': tfjs-backend-webgpu,
+  // 'cpu': tfjs-backend-wasm
```

Comment:
Nit: Since tfjs has its tfjs-backend-cpu backend, should we use 'wasm' explicitly for tfjs-backend-wasm?